product_uom_category_workflow_policy/models/product_uom_category.py

```diff
@@ -0,0 +1,156 @@
+# -*- coding: utf-8 -*-
+# Copyright 2020 OpenSynergy Indonesia
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import models, fields, api
+# from openerp.exceptions import Warning as UserError
+
+
+class ProductUomCategory(models.Model):
+    # jika 1 inherit tdk perlu buat _name
+    _name = "product.uom.categ"
+    _inherit = [
+        "product.uom.categ",
+        "base.workflow_policy_object",
+    ]
+
+    @api.depends(
+        "company_id",
+    )
+
+    # membuat fungsi untuk field compute -->
+    @api.multi
+    def _compute_policy(self):
+        _super = super(ProductUomCategory, self)
+        _super._compute_policy()
+
+    @api.multi
+    def _compute_company_id(self):
+        for document in self:
+            document.company_id = self.env.user.company_id
```

Comment:
Kalau tidak dipakai, dihapus saja

Reply:
Ini dari mas Andit.
Katanya just in case ada multi company